job_spec/ARIA_RAIDER.yml

```diff
@@ -0,0 +1,53 @@
+ARIA_RAIDER:
+  required_parameters:
+    - job_id
+  parameters:
+    job_id:
+      api_schema:
+        description: HyP3 Job Id for a INSAR_ISCE job that's successfully completed the DockerizedTopsApp step
+        type: string
+        format: uuid
+        example: 27836b79-e5b2-4d8f-932f-659724ea02c3
+    weather_model:
+      api_schema:
+        description: Weather model used to generate tropospheric delay estimations.
+        default: None
+        type: string
+        enum:
+          - None
+          - ERA5
+          - ERA5T
+          - GMAO
+          - HRES
+          - HRRR
+  cost_profiles:
+    DEFAULT:
+      cost: 1.0
+  validators: []
+  tasks:
+    - name: ''
+      image: ghcr.io/dbekaert/raider
+      command:
+        - ++process
+        - calcDelaysGUNW
+        - --bucket
+        - '!Ref Bucket'
+        - --bucket-prefix
+        - Ref::bucket_prefix
+        - --job-id
+        - Ref::job_id
```

Comment:
I believe we'll need to update RAiDER to take an additional parameter for this.

I had considered just passing the provided `job_id` to RAiDER as `--bucket-prefix`, and that *will* run through, but it would upload the competed products back to the previous INSAR_ISCE job-id prefix, not this job's prefix, in the hyp3 content bucket. So we need a way to get data from a different job-id prefix than the one we upload resulting products to.

Reply:
Thinking more about this, instead of taking a HyP3 Job ID; this could take a URL to a GUNW product, so it can work on HyP3 Jobs or Published products.

If we support running it on published products, we'll want RAiDER to update the version number and *not* clobber the already published GUWN in the archive

Reply:
Discussing it more with the ARIA team, we'll just focus on processing a previously run HyP3 job right now and circle back later to processing archived products.

Note: We want to be able to process either failed jobs or jobs that skipped the RAiDER step -- since failed jobs will not report a product URL, we'll want to stick with taking job_id as input.